src/rules/maintainability.ts

```diff
@@ -2,7 +2,7 @@
 // These rules make code maintenance easier
 // Reference https://palantir.github.io/tslint/rules/
 
-module.exports = {
+export const maintainabilityRules = {
```

Comment:
Named exports aren't necessary if there's always going to be a single export.  Just `export default [`.